src/main/java/com/microsoftopentechnologies/windowsazurestorage/beans/StorageAccountInfo.java

```diff
@@ -30,16 +30,22 @@ public class StorageAccountInfo {
     private String storageAccountKey;
 
     /**
-     * Azure stoarge blob end point url.
+     * Azure storage blob endpoint URL.
      */
     private String blobEndPointURL;
 
+    /**
+     * Azure storage CDN endpoint URL.
+     */
+    private String cdnEndPointURL;
+
     @DataBoundConstructor
     public StorageAccountInfo(String storageAccName, String storageAccountKey,
-                              String blobEndPointURL) {
+                              String blobEndPointURL, String cdnEndPointURL) {
```

Comment:
need to check if this is safe

Reply:
thanks - I'll wait till I hear back from you about this